x/ibc/20-transfer/types/expected_keepers.go

```diff
@@ -22,7 +22,6 @@ type ChannelKeeper interface {
 	SendPacket(ctx sdk.Context, channelCap *capability.Capability, packet channelexported.PacketI) error
 	PacketExecuted(ctx sdk.Context, packet channelexported.PacketI, acknowledgement []byte) error
 	ChanCloseInit(ctx sdk.Context, portID, channelID string, chanCap *capability.Capability) error
-	TimeoutExecuted(ctx sdk.Context, packet channelexported.PacketI) error
```

Comment:
`OnTimeoutPacket` no longer calls `TimeoutExecuted`. `TimeoutExecuted` is called by the ibc handler after running callback